packages/core-typings/src/IUser.ts

```diff
@@ -45,7 +45,35 @@ export type ILoginUsername =
 	  };
 export type LoginUsername = string | ILoginUsername;
 
-export interface IUserServices {
+export interface IOAuthUserServices {
+	google?: any;
```

Comment:
```suggestion
	google?: Record<string, any>;
```

:its_something:

Reply:
Reverted back to `any` as it was causing some TS issues.